gamemodes/terrortown/gamemode/shared/lang/en.lua

```diff
@@ -1297,3 +1297,12 @@ Keep in mind that these translations are made by the community. Feel free to con
 
 -- 2021-04-20
 L.none = "No Role"
+
+-- 2021-04-24
+L.karma_teamkill_tooltip = "Karma received by killing team member."
+L.karma_teamhurt_tooltip = "Karma received by damaging team member."
+L.karma_enemykill_tooltip = "Karma received by killing enemy."
+L.karma_enemyhurt_tooltip = "Karma received by damaging enemy."
+L.karma_cleanround_tooltip = "Karma received for clean round."
+L.karma_roundheal_tooltip = "Karma received at roundend."
+L.karma_unknown_tooltip = "Unknown Event."
```

Comment:
Is there a reason for this missing the Karma context?

I would prefer it to be:
```suggestion
L.karma_unknown_tooltip = "Unknown karma event."
```

Reply:
Not really. Just thought, that it would only be displayed in the karma tooltip and therefore is clear.
Anyway, after seeing how Tim implemented the tooltips I probably have to reduce the text by alot.
Like just Enemy damage, Enemy Kill etc.
Changed it for now.